packages/fxa-settings/src/components/BentoMenu/index.tsx

```diff
@@ -130,6 +131,20 @@ export const BentoMenu = () => {
                       <Localized id="bento-menu-vpn">Mozilla VPN</Localized>
                     </LinkExternal>
                   </li>
+                  <li>
+                    <LinkExternal
+                      data-testid="relay-link"
+                      href="https://relay.firefox.com/"
+                      className="block p-2 ltr:pl-6 rtl:pr-6 hover:bg-grey-100"
+                    >
+                      <div className={iconClassNames}>
+                        <img src={relayIcon} alt="" />
+                      </div>
+                      <Localized id="bento-menu-firefox-relay">
+                        Firefox Relay
+                      </Localized>
+                    </LinkExternal>
+                  </li>
```

Comment:
Looks like our Bento Menu tests could use some work and we don't even have `alt` text for other icons either? The heck. Do you mind filing a follow up issue for increasing our test coverage here, adding alt text, and making sure the alt text gets localized?

Reply:
👍🏼 Follow up issue created (#11136).